docs/api/process.md

```diff
@@ -215,6 +216,10 @@ Returns `Object`:
 Returns an object giving memory usage statistics about the entire system. Note
 that all statistics are reported in Kilobytes.
 
+### `process.getSystemProductType()`
+
+Returns `String` - Returns `server` if operating system is Windows Server. Otherwise, returns `desktop`.
```

Comment:
```suggestion
Returns `String` - Can be `server` or `desktop`. Returns `server` if operating system is Windows Server. Otherwise, returns `desktop`.
```

This should make the type definitions correct

Reply:
Thank you! Wasn't sure about the semantics. 😅